src/theory/strings/theory_strings_rewriter.cpp

```diff
@@ -970,8 +980,7 @@ Node TheoryStringsRewriter::rewriteStarRegExp(TNode node)
     return returnRewrite(node, node[0], "re-star-nested-star");
   }
   else if (node[0].getKind() == STRING_TO_REGEXP
-           && node[0][0].getKind() == CONST_STRING
-           && node[0][0].getConst<String>().isEmptyString())
+           && node[0][0].getKind() == CONST_STRING && Word::isEmpty(node[0][0]))
```

Comment:
We should have a method `Word::isWord(Node n)` that returns true for string/sequence constants and one for string/sequence regexps.

Reply:
Actually, we already do: `.isConst()` :)  That is the preferred way of checking this, indeed I should fix the kind comparison here.

Reply:
Btw, we will eventually need utility function for `isStringLike( TypeNode tn )` or something like that.